SPECS/php/php.spec

```diff
@@ -36,11 +27,10 @@
 %global with_sodium   1
 %global with_pspell   1
 %global with_lmdb     1
-
 Summary:        PHP scripting language for creating dynamic web sites
 Name:           php
 Version:        7.4.14
```

Comment:
Do we have a business justification for staying on the 7.4 branch? It's going EOL in two months, meaning we won't get any security patches from upstream. Upgrading to 8.1 gives us security support until Nov. 2024, which _should_ cover the lifespan of Mariner 2.0.

Source: https://www.php.net/supported-versions.php